PFCalEE/src/DetectorConstruction.cc

```diff
@@ -796,12 +858,12 @@ void DetectorConstruction::UpdateCalorSize(){
     m_CalorSizeXY=m_nSectors*m_sectorWidth;
   }
   else if (model_ == DetectorConstruction::m_2016TB ){
-    m_nSectors = 1;
-    m_sectorWidth = 160;
+    m_nSectors    = 1;
+    m_sectorWidth = CELL_SIZE_X * 2 *11;
     m_interSectorWidth = 0;
-    m_CalorSizeXY=m_nSectors*m_sectorWidth;
-    m_minRadius = 0;
-    m_maxRadius = m_CalorSizeXY/2.;
+    m_CalorSizeXY = m_sectorWidth;
```

Comment:
here I would keep the flexibility to do more than one wafer using:
m_CalorSizeXY = m_nSectors*m_sectorWidth

... but that can be changed later also.